pre-commit

```diff
@@ -28,7 +28,7 @@ for FILE in $FILES; do
     # Ensure the file still exists (i.e. is not being deleted).
     if [ -a $FILE ] ; then
       if [ ${FILE: -3} != ".sh" ] ; then
-        STAT="$(stat -f  "%A" $FILE)"
+        STAT="$(stat -c  "%a" $FILE)"
```

Comment:
This is not a cross platform solution...

alex: ~/dev/sites/drupal8alt.dev > STAT="$(stat -c  "%a" $FILE)"
stat: illegal option -- c
usage: stat [-FlLnqrsx] [-f format] [-t timefmt] [file ...]
alex: ~/dev/sites/drupal8alt.dev > STAT="$(stat -f  "%a" $FILE)"
 8.4.x  alex: ~/dev/sites/drupal8alt.dev > echo $STAT
1486812959
alex: ~/dev/sites/drupal8alt.dev > STAT="$(stat -f  "%A" $FILE)"
alex: ~/dev/sites/drupal8alt.dev > echo $STAT
644